builtin/settingtypes.txt

```diff
@@ -2067,6 +2067,13 @@ block_send_optimize_distance (Block send optimize distance) int 4 2 32767
 #    invisible blocks, so that the utility of noclip mode is reduced.
 server_side_occlusion_culling (Server-side occlusion culling) bool true
 
+#    At this distance the server will perform a simpler and cheaper occlusion check.
+#    Smaller values potentially improve performance, at the expense of temporarily visible
+#    rendering glitches (missing blocks).
+#    This is especially useful for very large viewing range (upwards of 500).
+#    Stated in mapblocks (16 nodes).
+block_cull_optimize_distance (Block cull optimize distance) int 20 2 32767
```

Comment:
```suggestion
#    Stated in MapBlocks (16 nodes).
block_cull_optimize_distance (Block cull optimize distance) int 20 2 2048
```

Reply:
I had copied from block_send_optimizer distance above. Should I fix those as well while I'm here?

Reply:
Sure.